src/conf.cpp

```diff
@@ -39,244 +39,138 @@ struct section {
     struct section *next;
 };
 
-ConfFile::~ConfFile()
+ConfFile::ConfFile(const char *filename)
+    : _filename(filename)
+    , _entry(nullptr)
+    , _last_section(nullptr)
+    , _n(0)
+    , _line(0)
```

Comment:
Since configuration files will be used on initialization, we could then add a call ConfParser::release_resources() that is called by the mainloop to unmap the file.